marconi-mamba/README.md

```diff
@@ -0,0 +1,35 @@
+marconi-mamba
```

Comment:
@koslambrou @raduom 
A draft README, should have enough info to build and test the client/server + rest-client script